Libraries/Components/Button.js

```diff
@@ -25,107 +26,220 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
+
+    [android:nextFocusLeft]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusLeft
+
+    @platform android, tv
    */
   nextFocusLeft?: ?number,
 
   /**
-   * TV next focus right (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates right. See
+    the [Android documentation][android:nextFocusRight].
+
+    [android:nextFocusRight]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusRight
+
+    @platform android, tv
    */
   nextFocusRight?: ?number,
 
   /**
-   * TV next focus up (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates up. See
+    the [Android documentation][android:nextFocusUp].
+
+    [android:nextFocusUp]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusUp
+
+    @platform android, tv
    */
   nextFocusUp?: ?number,
 
   /**
-   * Text to display for blindness accessibility features
+    Text to display for blindness accessibility features.
    */
   accessibilityLabel?: ?string,
 
   /**
-   * If true, disable all interactions for this component.
+    If `true`, disable all interactions for this component.
+
+    @default false
    */
   disabled?: ?boolean,
 
   /**
-   * Used to locate this view in end-to-end tests.
+    Used to locate this view in end-to-end tests.
    */
   testID?: ?string,
 |}>;
 
 /**
- * A basic button component that should render nicely on any platform. Supports
- * a minimal level of customization.
- *
- * <center><img src="img/buttonExample.png"></img></center>
- *
- * If this button doesn't look right for your app, you can build your own
- * button using [TouchableOpacity](docs/touchableopacity.html)
- * or [TouchableNativeFeedback](docs/touchablenativefeedback.html).
- * For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
- * Or, take a look at the [wide variety of button components built by the community](https://js.coach/react-native?search=button).
- *
- * Example usage:
- *
- * ```
- * import { Button } from 'react-native';
- * ...
- *
- * <Button
- *   onPress={onPressLearnMore}
- *   title="Learn More"
- *   color="#841584"
- *   accessibilityLabel="Learn more about this purple button"
- * />
- * ```
- *
+  A basic button component that should render nicely on any platform. Supports a
+  minimal level of customization.
+
+  If this button doesn't look right for your app, you can build your own button
+  using [TouchableOpacity](touchableopacity) or
+  [TouchableWithoutFeedback](touchablewithoutfeedback). For inspiration, look at
+  the [source code for this button
+  component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
```

Comment:
@ayushjainrksh, sorry to block this PR again on a minor detail, but could you please go through all links and fix the ones that are causing awkward line breaks? (e.g. using references like you did elsewhere)

Reply:
@motiz88 Ahh, sorry it went unnoticed. Fixed it now :)